BAZEL.md

```diff
@@ -370,13 +370,18 @@ detailed information.
 - Execute a test with a specific name
 
     ```
-    bazel test //ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_JdbcLedgerDaoSpec.scala --test_arg=-t --test_arg="JDBC Ledger DAO should be able to persist and load contracts without external offset"
-    ```    
+    bazel test 
+    //ledger/sandbox:sandbox-scala-tests_test_suite_src_test_suite_scala_com_digitalasset_platform_sandbox_stores_ledger_sql_JdbcLedgerDaoSpec.scala 
+    --test_arg=-t 
+    --test_arg="JDBC Ledger DAO should be able to persist and load contracts without external offset"
+    ```
```

Comment:
Perhaps add ` \` to the end of each line (aside from the last one) to signal continuation?

Reply:
Agreed, making the continuations more obvious would be good. `\` line endings and maybe also indenting the continued lines a bit.

Reply:
updated to include the slashes